### PR TITLE
Aim integration tests search aligned metrics 

### LINKED ISSUE #404: Add integration tests for Aim API SearchAlignedMetrics endpoint

### PR DESCRIPTION
**Pull request by fabiovincenzi**
fixes #404

### CODE REVIEW

File: pkg/api/aim/request/metric.go [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+type AlignedMetricRunRequest struct {
+	ID     string                      `json:"run_id"`
+	Traces []AlignedMetricTraceRequest `json:"traces"`
+}
+
+type AlignedMetricTraceRequest struct {
+	Context fiber.Map `json:"context"`
+	Name    string    `json:"name"`
+	Slice   [3]int    `json:"slice"`
+}
```
Comment (dsuhinin) [inline pkg/api/aim/request/metric.go:22]: Can you please add comments here?

File: pkg/api/aim/request/metric.go [hunk 2]
```diff
@@ -1,5 +1,26 @@
 package request
 
+import (
+	"github.com/gofiber/fiber/v2"
+)
+
+// GetAlignedMetricRequest is a request object for `GET /mlflow/metrics/align` endpoint.
+type GetAlignedMetricRequest struct {
+	AlignBy string                    `json:"align_by"`
+	Runs    []AlignedMetricRunRequest `json:"runs"`
```
Comment (dsuhinin) [inline pkg/api/aim/request/metric.go:10]: can you please just move `AlignBy` to be underneath `Runs`?

File: pkg/api/aim/request/metric.go [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+
+type AlignedMetricRunRequest struct {
+	ID     string                      `json:"run_id"`
+	Traces []AlignedMetricTraceRequest `json:"traces"`
+}
+
+type AlignedMetricTraceRequest struct {
+	Context fiber.Map `json:"context"`
+	Name    string    `json:"name"`
+	Slice   [3]int    `json:"slice"`
```
Comment (dsuhinin) [inline pkg/api/aim/request/metric.go:21]: is it really only 3 items size?

**Review by dsuhinin — CHANGES_REQUESTED** [review]
small adjustments.

File: pkg/api/aim/request/metric.go (hunk 3 above)
Comment (fabiovincenzi) [reply pkg/api/aim/request/metric.go:21]: I'm not really sure, I used 3 since I saw it on `SearchAlignedMetrics` endpoint, but anyway now I changed it.

Comment (dsuhinin) [reply pkg/api/aim/request/metric.go]: no no, I just curious why only 3?

File: pkg/api/aim/request/metric.go [hunk 4]
```diff
@@ -1,5 +1,28 @@
 package request
 
+import (
+	"github.com/gofiber/fiber/v2"
+)
+
+// GetAlignedMetricRequest is a request object for `GET /mlflow/metrics/align` endpoint.
```
Comment (dsuhinin) [inline pkg/api/aim/request/metric.go:7]: this request object is for `aim` and not for `mlflow`. can you please change?

File: pkg/api/aim/request/metric.go (hunk 3 above)
Comment (dsuhinin) [reply pkg/api/aim/request/metric.go]: @jgiannuzzi is there a reason to have array of 3 elements?

Comment (dsuhinin) [reply pkg/api/aim/request/metric.go]: as I can see from the code - not really.

**Review by dsuhinin — APPROVED** [review]
nice!

**Pull request merged**